src/atoms/dataEntry/CheckboxField/CheckboxField.js

```diff
@@ -22,9 +22,9 @@ const CheckboxField = ({
   meta = {},
   ...rest
   }: CheckboxFieldProps) => {
-  const { error, touched } = meta;
-  const { name, value, onChange, onBlur, onFocus } = input;
-  const hasError = !!error && !!touched;
+  const { submitError, dirtySinceLastSubmit, error, touched } = meta;
+  const { name, value, onChange, onFocus, onBlur } = input;
+  const hasError = (!!error || !!submitError) && !!touched && !dirtySinceLastSubmit;
```

Comment:
I think you should create common function `checkHasError = (meta) => boolean`.

Reply:
@vorobeez fixed